docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/ShoppingCartLoader.scala

```diff
@@ -91,7 +92,7 @@ class ShoppingCartServiceImpl(
 // #shopping-cart-service-impl
     {
   // #shopping-cart-entity-ref
-  def entityRef(id: String): EntityRef[ShoppingCart.Command] = {
+  def entityRef(id: String): EntityRef[Command] = {
```

Comment:
I think that using `ShoppingCart.Command` instead of `Command` is clearer for users (better differentiate between a _generic_ command and the `ShoppingCart.Command`). Same as for `Summary` a few lines below.

Reply:
This happened because `ShoppingCart` is not the main wrapper object, but `ShoppingCartExamples`. See my comment in https://github.com/lagom/lagom/pull/2435#discussion_r347597510

I will move things around and try to improve on this.